modules/issue_tracker/php/issue_tracker.class.inc

```diff
@@ -115,7 +115,12 @@ class Issue_Tracker extends \NDB_Menu_Filter_Form
             $assignees[$a_row['UserID']] = $a_row['Real_name'];
         }
 
-        $modules = \Module::getActiveModules($db);
+        $mod_descriptors = \Module::getActiveModules($db);
```

Comment:
I'll be a bit picky here but the variable name is not quite accurate. The returned value of `\Module::getActiveModules` is an array of `\Module` instances. It should probably be `$modules` but I understand that the intent is to use $modules to store the results of the next foreach loop. `$active_modules` would do it.

Suggestion: You could make 2 hits one stone by using array_reduce.
```
$modules = array_reduce(\Module::getActiveModules($db), function ($carry, $module) {
    ...
}, [])
```